### ISSUE TITLE
Keyboard accessibility for ng-material-multilevel-menu

### ISSUE THREAD

**Issue by harish-kp-reddy** [ShankyTiwari/ng-material-multilevel-menu#204]
This menu lacks the support of Keyboard accessibility. in the project we need this feature please help me.

**Comment by ShankyTiwari** [comment]
Hey @harish-kp-reddy I don't have any plans for that as of now as It would require a good amount of time to add that feature.

Thanks
